pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -249,17 +248,17 @@ func validateVirtualMachineInstanceSpecVolumeDisks(field *k8sfield.Path, spec *v
 }
 
 func validateNetworksMatchInterfaces(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) (causes []metav1.StatusCause) {
-	networkNameMap := vmispec.IndexNetworkSpecByName(spec.Networks)
+	//networkNameMap := vmispec.IndexNetworkSpecByName(spec.Networks)
 
 	// Make sure the port name is unique across all the interfaces
-	portForwardMap := make(map[string]struct{})
+	//portForwardMap := make(map[string]struct{})
 
 	// Validate that each interface has a matching network
 	for idx, iface := range spec.Domain.Devices.Interfaces {
 
-		networkData, networkExists := networkNameMap[iface.Name]
+		//networkData, networkExists := networkNameMap[iface.Name]
 
-		causes = append(causes, validatePortConfiguration(field, networkExists, &networkData, iface, idx, portForwardMap)...)
+		//causes = append(causes, validatePortConfiguration(field, networkExists, &networkData, iface, idx, portForwardMap)...)
```

Comment:
Shouldn't the commented line be removed?

Reply:
Looks like leftovers, thanks.
Done.